beacon-chain/rpc/beacon/attestations.go

```diff
@@ -132,17 +132,14 @@ func (bs *Server) ListIndexedAttestations(
 	epoch := helpers.SlotToEpoch(bs.GenesisTimeFetcher.CurrentSlot())
 	switch q := req.QueryFilter.(type) {
 	case *ethpb.ListIndexedAttestationsRequest_TargetEpoch:
-		atts, err = bs.BeaconDB.Attestations(ctx, filters.NewFilter().SetTargetEpoch(q.TargetEpoch))
```

Comment:
This changes the whole purpose of the method, we want to use this method to retrieve attestations from the db.